.pre-commit-config.yaml

```diff
@@ -43,6 +42,12 @@ repos:
   rev: v2.16.3
   hooks:
   - id: pylint
+    args:
+    - '--load-plugins=pylint_django'
+    - '--django-settings-module=mysite.settings'
```

Comment:
No need to quote values like this. Unquote it.

Reply:
Addressed  6fe8f02d4759fadd4bb75824fb41ca67c164451d

Reply:
Marking this as resolved.